crates/kornia-imgproc/src/calibration/distortion.rs

```diff
@@ -222,4 +583,209 @@ mod tests {
 
         Ok(())
     }
+
+    // helper function
+    fn realistic_intr() -> CameraIntrinsic {
+        CameraIntrinsic {
+            fx: 612.3,
+            fy: 610.8,
+            cx: 320.1,
+            cy: 241.7,
+        }
+    }
+
+    // helper function
+    fn realistic_dist() -> PolynomialDistortion {
+        PolynomialDistortion {
+            k1: -0.321,
+            k2: 0.124,
+            k3: -0.012,
+            k4: 0.210,
+            k5: -0.015,
+            k6: 0.001,
+            p1: 0.00042,
+            p2: -0.00031,
+        }
+    }
+
+    // helper function
+    fn strong_dist() -> PolynomialDistortion {
+        PolynomialDistortion {
+            k1: -0.7,
+            k2: 0.35,
+            k3: -0.05,
+            k4: 0.6,
+            k5: 0.05,
+            k6: -0.01,
+            p1: 0.0008,
+            p2: -0.0006,
+        }
+    }
+
+    // helper function
+    fn no_dist() -> PolynomialDistortion {
+        PolynomialDistortion {
+            k1: 0.,
+            k2: 0.,
+            k3: 0.,
+            k4: 0.,
+            k5: 0.,
+            k6: 0.,
+            p1: 0.,
+            p2: 0.,
+        }
+    }
+
+    // helper function
+    fn approx(a: f64, b: f64, tol: f64) -> bool {
+        (a - b).abs() <= tol
+    }
+
+    // helper function
+    fn forward_dist_norm(x: f64, y: f64, d: &PolynomialDistortion) -> (f64, f64) {
```

Comment:
Why don’t you implement as part of the public api this already with testing as well